### PR TITLE
Support scalyr log shipping

### LINKED ISSUE #181: Scalyr agent for log shipping
* Docker container logs
* journald system logs

### PR DESCRIPTION
**Pull request by mohabusama**
Resolve #181

### CODE REVIEW

File: cluster/userdata-master.yaml [hunk 1]
```diff
@@ -1126,105 +1126,152 @@ write_files:
...
                 mountPath: /mnt/containers
                 readOnly: true
-              - name: jobfiles
-                mountPath: /appd/agent/monitors/analytics-agent/conf/job
+              - name: scalyr-logs
+                mountPath: /mnt/scalyr-logs
+                readOnly: false
+              - name: scalyr-config
+                mountPath: /mnt/scalyr-config
+                readOnly: false
```
Comment (szuecs) [inline cluster/userdata-master.yaml]: Do you write into your config ?

Comment (mohabusama) [inline cluster/userdata-master.yaml]: Yes, the ``log-watcher`` updates the scalyr-config ``agent.json`` which is then consumed and detected by ``scalyr-agent`` container below.

File: cluster/userdata-master.yaml [hunk 2]
```diff
@@ -1126,105 +1126,152 @@ write_files:
...
+                mountPath: /var/log/journal
+                readOnly: true
+
               resources:
                 limits:
                   cpu: 200m
                   memory: 500Mi
                 requests:
                   cpu: 100m
                   memory: 100Mi
```
Comment (hjacobs) [inline cluster/userdata-master.yaml]: sounds like a lot of memory..

**Comment by hjacobs** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Pull request merged**